.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+### Expected behavior
```

Comment:
Can we file this as a separate PR ? Also we need to talk internally if we want to open up the issue queue.